usaspending_api/disaster/v2/views/object_class/object_class_result.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass, field
+from dataclasses_json import dataclass_json
+from typing import Dict
+
+from usaspending_api.common.data_classes import Pagination
+from usaspending_api.disaster.v2.views.data_classes import Collation, Element
+
+
+class ObjectClass(Element):
+    """Renaming the original generic object to clearly be TAS"""
+
+
+class MajorClass(Collation):
+    """Renaming the original generic object to clearly be Federal Account"""
+
+
+@dataclass_json
+@dataclass
+class ObjectClassResults:
```

Comment:
The only change between this and `FedAcctResults` appears to be removing the `total_budgetary_resources` from the rollup (and dropping it in finalize). There is already an example in the federal_account/loans/ endpoint of dealing with that unnecessary attribute.

Seems like trivial changes to re-define the dataclass. Are you planning to duplicate again for Agencies as well?

Reply:
The other big difference are the differently named data classes. I would love to refactor both classes to reconcile them more cleanly, but I had assumed you would interpret that as being too high risk this late in the game.

Reply:
Could duplicate the code above,

```python
class ObjectClassResults(FedAcctResults):
    """Renaming the original object to clearly be ObjectClass"""
```

Reply:
I'd still have to redefine the functions to use the object class named components instead of the federal account named ones.

Reply:
Technically you wouldn't since the data classes are identical. Python duck-typing. I'm not going to hold up the PR over this.